src/main/java/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,66 @@
+package duke.commands;
+
+import duke.exceptions.DukeWriteFailException;
+import duke.tasks.Deadline;
+import duke.tasks.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents the command 'deadline'.
+ */
+public class DeadlineCommand extends Command {
+    String taskName;
+    String dateTime;
+
+    /**
+     * Creates a command to create a Deadline task.
+     * @param taskName name of deadline.
+     * @param dateTime date and time of deadline.
+     */
+    public DeadlineCommand(String taskName, String dateTime) {
+        this.taskName = taskName;
+        this.dateTime = dateTime;
+    }
+
+    /**
+     * Method to check if command is the exit command.
```

Comment:
Same here. Please update the method comments. It should start with simple present e.g. Checks